package.json

```diff
@@ -18,11 +18,16 @@
     "*.js",
     "test"
   ],
+  "nyc": {
+    "require": ["babel-register"],
+    "reporter": ["text", "html"]
+  },
   "scripts": {
-    "test": "istanbul cover --report cobertura _mocha -- --compilers js:babel-register",
-    "pretest": "godaddy-js-style-lint *.js ./test",
-    "prepublish": "npm run build",
-    "build": "babel *.js -d lib"
+    "build": "babel *.js -d lib",
+    "lint": "eslint --fix *.js test",
```

Comment:
The other argument @3rd-Eden is that the `eslint-godaddy-react` binaries have 💩 💩 💩 support for Windows due to how `which` works cross platform. Keeping it this way make contribution more accessible to Windows folks.